src/indicators/ipc_food_insecurity/update_food_insecurity.R

```diff
@@ -7,6 +9,17 @@ box::use(./utils/map_food_insecurity)
 box::use(./utils/info_food_insecurity)
 
 box::use(../../alerts/generate_signals[generate_signals])
+box::use(../../utils/logger)
+
+logger$configure_logger()
+
+test <- as.logical(Sys.getenv("TEST", unset = FALSE))
```

Comment:
A good point. I was thinking that a future PR could have some further refactoring to avoid the issues with `test_filter` that we talked about, so that it doesn't potentially fail if `test_filter` isn't set. I'll create an issue and pick that up in future work!